tools/devops/automation/templates/packages/publish-html.yml

```diff
@@ -62,16 +70,34 @@ steps:
       Write-Host "##vso[task.setvariable variable=$envVarName]$url"
     }
   displayName: 'Create API from stable diff gists'
+  enabled: false
```

Comment:
The gist is giving me issues with the token, will work on it in a diff PR, but I want the pkgs in the comment asap.